pkg/util/net.go

```diff
@@ -0,0 +1,22 @@
+package util
```

Comment:
It looks like these are only used in one place. Does this need to be it's own package?